task/3scale-toolbox-import/0.1/tests/run.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-import-openapi-sample
+spec:
+  tasks:
+    - name: 3scale-toolbox-import
+      params:
+        - name: CONTEXT_DIR
+          value: shared-workspace
+        - name: PATH_OPENAPI
+          value: swagger.json
+        - name: PRIVATE_BASE_URL
+          value: 'http://sample.svc.cluster.local:8080'
+        - name: TARGET_SYSTEM_NAME
+          value: sample_api
+        - name: DESTINATION
+          value: 3scale28
+      runAfter:
+        - git-clone
+      taskRef:
+        kind: Task
+        name: 3scale-toolbox-import
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+    - name: git-clone
+      params:
+        - name: url
+          value: 'https://github.com/rafamqrs/camel-quarkus-openapi.git'
+        - name: revision
+          value: main
+        - name: submodules
+          value: 'true'
+        - name: depth
+          value: '1'
+        - name: sslVerify
+          value: 'true'
+        - name: subdirectory
+          value: shared-workspace
+        - name: deleteExisting
+          value: 'true'
+        - name: verbose
+          value: 'true'
+        - name: gitInitImage
+          value: >-
+            registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:afc5d3f9efe26c7042635d43b8ffd09d67936e3d0b6b901dc08a33e20313d361
```

Comment:
This image might not be public. Any specific reason for choosing this image in place of the one from gcr?

Reply:
Removed this image and the task git-clone